src/generated/client/apis/TimebankApi.ts

```diff
@@ -129,11 +129,35 @@ export class TimebankApi extends runtime.BaseAPI {
 
     /**
      */
-    async timebankControllerGetTotal(requestParameters: TimebankControllerGetTotalRequest): Promise<Array<any>> {
+    async timebankControllerGetTotal(requestParameters: TimebankControllerGetTotalRequest): Promise<Array<TimeEntryTotalDto>> {
         const response = await this.timebankControllerGetTotalRaw(requestParameters);
         return await response.value();
     }
 
+    /**
+     */
+    async timebankControllerSyncWorkTimeRaw(): Promise<runtime.ApiResponse<string>> {
+        const queryParameters: any = {};
+
+        const headerParameters: runtime.HTTPHeaders = {};
+
+        const response = await this.request({
+            path: `/timebank/sync`,
+            method: 'POST',
+            headers: headerParameters,
+            query: queryParameters,
+        });
+
+        return new runtime.TextApiResponse(response) as any;
+    }
```

Comment:
Why any type?

Reply:
generate by openapi tool, but I think the yaml is outdated so its regenerated